src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/LanguageServices/ActiveEditorContextTrackerTests.cs

```diff
@@ -236,7 +236,7 @@ public void GetProjectName_WhenActiveIntellisenseProjectContextIdSetToNull_Retur
 
         private static ActiveEditorContextTracker CreateInstance()
         {
-            return new ActiveEditorContextTracker();
+            return new ActiveEditorContextTracker((UnconfiguredProject)null);
```

Comment:
Could we add a test that uses two projects and verifies that they get different trackers?

Reply:
Unlike CPS, we don't have integration tests at that level, we need their implementation assemblies to be able to correctly setup "scopes" that would mimic this bug.

Reply:
Given our lack of VS tests right now, and for the test verification that I'm reviving above - I'll figure out a way to get access to their implementation binaries so that we can do this sort of thing.